src/shadowbox/model/outline_error.ts

```diff
@@ -0,0 +1,21 @@
+class OutlineError extends Error {
```

Comment:
Please add the header license to this file. I also wonder if it should be in the `infrastructure` directory, and whether the filename should be `errors.ts` (that is how it is currenlty defined in the server manager).

Reply:
Done.  Kept in `model` pending Vini's review